runsc/cmd/chroot.go

```diff
@@ -59,6 +59,22 @@ func pivotRoot(root string) error {
 	return nil
 }
 
+func copyFile(dst, src string) (int64, error) {
```

Comment:
The caller is not using the number of bytes read and this function is guaranteed to copy all or fail, I think it could only return `error`.

Reply:
Make sense. Will fix.